fluent-bundle/src/bundle.rs

```diff
@@ -43,10 +49,12 @@ pub struct Message<'m> {
 ///     .expect("Failed to add FTL resources to the bundle.");
 ///
 /// let mut args = HashMap::new();
-/// args.insert("name", FluentValue::from("Rustacean"));
+/// args.insert("name".to_string(), FluentValue::from("Rustacean"));
 ///
-/// let (value, _) = bundle.format("intro", Some(&args))
-///     .expect("Failed to format a message.");
+/// let msg = bundle.get_message("intro").expect("Message exists");
+/// let mut errors = vec![];
+/// let pattern = msg.value.expect("Message has a value");
```

Comment:
Same as in examples, please use a proper `if let Some(value) = ...`.

Reply:
Also, I can't comment on the exact line, but this docstring mentions `format` and `compound` a few times.